osu.Game/OsuGame.cs

```diff
@@ -64,14 +64,16 @@ namespace osu.Game
     /// </summary>
     public class OsuGame : OsuGameBase, IKeyBindingHandler<GlobalAction>
     {
+        public const float SCREEN_OFFSET_RATIO = 0.125f;
```

Comment:
Is there a reason this is higher than it used to be on `master`? It looks jarring now.

Reply:
Ah I miscalculated the ratio, `0.05` sounds correct and inline with master.